src/components/Button/index.tsx

```diff
@@ -366,35 +367,17 @@ export const AddSWPRToMetamaskButton = styled(Base)<{ active?: boolean }>`
   box-shadow: ${props => (props.active ? '0px 0px 42px rgba(165, 58, 196, 0.35)' : 'none')};
 `
 
-const MoreButton = styled.button`
-  position: relative;
-  display: block;
-  padding: 6px 8px;
-  margin: 12px auto 0;
-  font-weight: 600;
-  font-size: 10px;
-  line-height: 12px;
-  letter-spacing: 0.12em;
-  text-transform: uppercase;
-  color: ${({ theme }) => theme.text5};
-  background: none;
-  border: none;
-  cursor: pointer;
-
-  img {
-    margin-left: 10px;
-  }
-`
-
-interface ShowMoreButtonProps {
-  children: ReactNode
-  onClick: () => void
-  isOpen: boolean
-}
-
-export const ShowMoreButton = ({ children, onClick, isOpen }: ShowMoreButtonProps) => (
-  <MoreButton onClick={onClick}>
-    {children}
-    <img src={arrowIcon} alt="arrow down" style={{ transform: isOpen ? 'rotate(180deg)' : 'rotate(0)' }} />
-  </MoreButton>
-)
+export const StyledButtonsArray = [
+  Base,
+  ButtonPrimary,
+  ButtonSecondary,
+  ButtonGrey,
+  ButtonInvisbile,
+  ButtonDark1,
+  ButtonPurple,
+  ButtonDark2,
+  ButtonOutlined,
+  ButtonEmpty,
+  ButtonWhite,
```

Comment:
Button White seems to be not working
![Screenshot 2022-06-29 at 23 26 31](https://user-images.githubusercontent.com/102727631/176547823-9fae7320-b299-48ce-9065-9574a5e42022.png)

Reply:
yeah its some shitty unusued button just deleted it